### PR TITLE
Render logged in user profile on root page

### PR DESCRIPTION
**Pull request by christophehenry**
#61

### CODE REVIEW

**Comment by christophehenry** [comment]
Halp! :(

**Comment by jaywink** [comment]
You need to fix the tests to match the new functionality. Test `test_home_view_rendered_without_root_profile` in https://travis-ci.org/jaywink/socialhome/jobs/207769585#L734 is checking that `home.html` is rendered, but now `profile.html` is rendered, as the test is using a logged in user.

These test adjustments might work? There is a new test and also checking that correct profile is in context. I want to test everything in Socialhome, so you'll need to learn py.test and Django testing.. :)

    @pytest.mark.usefixtures("db")
    class TestRootProfile(object):
        @pytest.mark.usefixtures("client", "settings")
        def test_home_view_rendered_without_root_profile(self, client, settings):
            settings.SOCIALHOME_ROOT_PROFILE = None
            response = client.get("/")
            assert response.templates[0].name == "pages/home.html"
    
        @pytest.mark.usefixtures("admin_client", "settings")
        def test_logged_in_profile_view_rendered_without_root_profile(self, admin_client, settings):
            settings.SOCIALHOME_ROOT_PROFILE = None
            response = admin_client.get("/")
            assert response.templates[0].name == "streams/profile.html"
            assert response.context["profile"].user.username == "admin"
    
        @pytest.mark.usefixtures("client", "admin_client", "settings")
        def test_home_view_rendered_with_root_profile(self, client, admin_client, settings):
            settings.SOCIALHOME_ROOT_PROFILE = "admin"
            # Set admin profile visibility, otherwise it will just redirect to login
            Profile.objects.filter(user__username="admin").update(visibility=Visibility.PUBLIC)
            response = client.get("/")
            assert response.templates[0].name == "streams/profile.html"
            assert response.context["profile"].user.username == "admin"

**Comment by christophehenry** [comment]
Your adjusment work. Though I do not clearly understand why, yet.

File: socialhome/content/tests/test_views.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from types import SimpleNamespace
```
Comment (jaywink) [inline socialhome/content/tests/test_views.py]: This import is unused now I think?

**Review by jaywink — CHANGES_REQUESTED** [review]

File: socialhome/content/tests/test_views.py (hunk 1 above)
Comment (christophehenry) [reply socialhome/content/tests/test_views.py]: Yes, it is. I forgot that one.

**Review by jaywink — APPROVED** [review]
Awesome, thanks!

**Pull request merged**